lib/tasks/developer_digest.rake

```diff
@@ -0,0 +1,37 @@
+namespace :developer_digest do
+  desc "email subscribed businesses about new developers in the last day"
+  task daily: :environment do
+    businesses = Business.daily_developer_notifications
+    developers = Developer.where("created_at > '#{1.day.ago}'")
```

Comment:
```suggestion
    developers = Developer.where(created_at: Date.yesterday..Date.current)
```

This creates a concrete time span instead of basing it off of when the task was actually run. If the time the task run changes we could miss a developer otherwise.